common/src/main/java/com/wynntils/models/ingredients/IngredientModel.java

```diff
@@ -23,8 +24,8 @@ public class IngredientModel extends Model {
             ChatFormatting.AQUA, 3);
     private final IngredientInfoRegistry ingredientInfoRegistry = new IngredientInfoRegistry();
 
-    public IngredientModel() {
-        super(List.of());
+    public IngredientModel(WynnItemModel wynnItem) {
```

Comment:
Why do you need this? (Same for the RewardModel)

Reply:
Oh, yeah! I forgot to comment for this here. I planned to, though :). Anyways, `xInfoRegistry` (Ingredient/Charm/Tome) uses `WynnItem` models methods for converting item material to item stack. `IngredientModel`'s already been depending on `WynnItem` implicitly, by only loading ingredients after loading material conversions and item obtain data. https://github.com/Wynntils/Artemis/pull/2424/commits/72072879fe918bea2d3696df81fc7256bc444fa1 should clear this up even further.

Reply:
As to why this was fixed now, it's because I saw a log where this happened. This is a really rare race case, so it's not been a problem before, but it should be stable now.

Reply:
I see. Yeah, that was a tricky one. Thanks for adding the description.